### PR TITLE
Add RANK to the ranked lists

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
See #1022 for a discussion.
- I added a RANK column to the lists tagged as ranked.

- In Joo's list, I removed the tag "ranked" as I could not find any implications of ordering in the source. Moreover, the author states that the list is taken from the Leipzig-Jakarta list and it is provided in the paper in alphabetical order. Initially, the list as it is now in concepticon was taken from the respective Lexibank repository. If everyone agrees on that I'll reorder it correspondingly.

- I also refined a few mappings in Peust-2013-54.

### CODE REVIEW

**Comment by Kristina-Pianykh** [comment]
Could I ask you to review this PR, @LinguList if you have time, since we were discussing this issue with you earlier?

**Review by LinguList — APPROVED** [review]
Nice, one review is probably enough here, but @AnnikaTjuka could also have a quick look, as this concerns the norare-part to some degree. I am not sure if we should  have the "ratings" tag on Vejdemo, since their stability rates are not necessarily the same as we'd use for ratings (here, Peust wit hthe 1-6 scale is more of a rate). We should maybe review "ratings" in our tags later independently. Many thanks!

**Comment by AnnikaTjuka** [comment]
Thanks for updating the ranks, @Kristina-Pianykh ! I actually wasn't aware of the Vejdemo-2016 list yet. I'm guessing that they added the ratings on imageability, concreteness etc. from secondary sources. So for example, the Brysbaert et al concreteness ratings are already included as a separate data set in NoRaRe. I'd therefore suggest that we do not add the tag "ratings" and I'll check out the sources they used so that we can add the original data sets to NoRaRe.

**Review by LinguList — APPROVED** [review]
Ah, sorry, @Kristina-Pianykh and @AnnikaTjuka, I did not know we have real ratings in the data of Vejdemo, so I'd rather keep them there, as they published this along with the list, right? So they provide a list that has ratings. I thought there were only the stability scores, my error!

**Comment by AnnikaTjuka** [comment]
Ok, sure. I'll still see whether they use some secondary sources that we can add to NoRaRe. @Kristina-Pianykh could you update the column names in the list to all caps?

**Comment by LinguList** [comment]
Yes, @AnnikaTjuka, nice if you check their sources!

**Comment by Kristina-Pianykh** [comment]
As far as Vejdemo’s list is concerned, some data are taken from other sources (such as words class and frequency (Pagel at al 2007), age of acquisition (Monaghan 2014), arousal (Warriner et al. 2013), imaginability (Cortese & Fugett 2004) and senses (based on Wordnet English lexical database). Other data arises in the study itself (mutual information, synonyms).

I will then update the column names tomorrow when I can use my laptop.

File: concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json [hunk 1]
```diff
@@ -44,63 +44,67 @@
                     },
                     {
                         "datatype": "float", 
-                        "name": "Rate"
+                        "name": "RATE"
                     },
                     {
                         "datatype": "float",
-                        "name": "Synonyms_average"
+                        "name": "SYNONYMS_AVERAGE"
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json:51]: SYNONYMS_AVERAGE = SYNONYMS_MEAN

File: concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json [hunk 2]
```diff
@@ -44,63 +44,67 @@
...
                     },
                     {
                         "datatype": "float", 
-                        "name": "Imageability"
+                        "name": "IMAGEABILITY"
                     },
                     {
                         "datatype": "float",
-                        "name": "BrysbaertEtAl_concreteness"
+                        "name": "BRYSBAERT_CONCRETENESS"
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json:83]: BRYSBAERT_CONCRETENESS = CONCRETENESS

File: concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json [hunk 3]
```diff
@@ -44,63 +44,67 @@
...
                     },
                     {
                         "datatype": "float",
-                        "name": "AgeOfAcquisition"
+                        "name": "AGE_OF_ACQUISITION"
                     },
                     {
                         "datatype": "string", 
-                        "name": "WordClass"
+                        "name": "WORDCLASS"
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Vejdemo-2016-167.tsv-metadata.json:103]: WORDCLASS = POS

**Review by AnnikaTjuka — COMMENTED** [review]
I'd like to propose some small changes in the column names in the Vejdemo-2016 list so that they are consistent with other column names in NoRaRe.

**Review by AnnikaTjuka — APPROVED** [review]
Many thanks, @Kristina-Pianykh !

**Pull request merged**